pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -86,6 +88,16 @@ func add(mgr manager.Manager, r *ReconcileOperConfig) error {
 		return err
 	}
 
+	// Likewise for the ConfigMap reconciler
+	c, err = controller.New("configmap-controller", mgr, controller.Options{Reconciler: r.configMapReconciler})
+	if err != nil {
+		return err
+	}
+	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForObject{})
```

Comment:
I'm not certain if it will work, but I'd like to see this use `hander.EnqueueRequestForOwner{}`. That way we don't try and reconcile *every* configmap update.